<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="2" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-03-08T15:58:58 IST" name="Suite" finished-at="2023-03-08T16:00:25 IST" duration-ms="87187">
    <groups>
    </groups>
    <test started-at="2023-03-08T15:58:58 IST" name="Test" finished-at="2023-03-08T16:00:25 IST" duration-ms="87187">
      <class name="com.orangehrm.test_cases.MyInforPageTest">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:com.orangehrm.test_cases.MyInforPageTest@bae7dc0]" started-at="2023-03-08T16:00:02 IST" name="openBrowser" finished-at="2023-03-08T16:00:06 IST" duration-ms="3905" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="verifyMyInfo()[pri:0, instance:com.orangehrm.test_cases.MyInforPageTest@bae7dc0]" started-at="2023-03-08T16:00:06 IST" name="verifyMyInfo" finished-at="2023-03-08T16:00:25 IST" duration-ms="18765" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyMyInfo -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.orangehrm.test_cases.MyInforPageTest@bae7dc0]" started-at="2023-03-08T16:00:25 IST" name="closeBrowser" finished-at="2023-03-08T16:00:25 IST" duration-ms="169" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.orangehrm.test_cases.MyInforPageTest -->
      <class name="com.orangehrm.test_cases.LoginTest_TC1">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:00 IST" name="openBrowser" finished-at="2023-03-08T15:59:06 IST" duration-ms="6339" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="loginToApplication(java.lang.String,java.lang.String)[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:06 IST" name="loginToApplication" data-provider="supplyData" finished-at="2023-03-08T15:59:16 IST" duration-ms="9841" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:16 IST" name="closeBrowser" finished-at="2023-03-08T15:59:16 IST" duration-ms="194" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:16 IST" name="openBrowser" finished-at="2023-03-08T15:59:21 IST" duration-ms="5310" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="loginToApplication(java.lang.String,java.lang.String)[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:21 IST" name="loginToApplication" data-provider="supplyData" finished-at="2023-03-08T15:59:31 IST" duration-ms="9298" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[RandomUser]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[abc123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.orangehrm.test_cases.LoginTest_TC1.loginToApplication(LoginTest_TC1.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:31 IST" name="closeBrowser" finished-at="2023-03-08T15:59:31 IST" duration-ms="167" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:31 IST" name="openBrowser" finished-at="2023-03-08T15:59:38 IST" duration-ms="6451" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="loginToApplication(java.lang.String,java.lang.String)[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:38 IST" name="loginToApplication" data-provider="supplyData" finished-at="2023-03-08T15:59:47 IST" duration-ms="9389" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[User2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[xyz123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.orangehrm.test_cases.LoginTest_TC1.loginToApplication(LoginTest_TC1.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:47 IST" name="closeBrowser" finished-at="2023-03-08T15:59:48 IST" duration-ms="170" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:48 IST" name="openBrowser" finished-at="2023-03-08T15:59:53 IST" duration-ms="4965" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method signature="loginToApplication(java.lang.String,java.lang.String)[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T15:59:53 IST" name="loginToApplication" data-provider="supplyData" finished-at="2023-03-08T16:00:02 IST" duration-ms="9222" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[User3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[lmno123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.orangehrm.test_cases.LoginTest_TC1.loginToApplication(LoginTest_TC1.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.orangehrm.test_cases.LoginTest_TC1@76c3e77a]" started-at="2023-03-08T16:00:02 IST" name="closeBrowser" finished-at="2023-03-08T16:00:02 IST" duration-ms="217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.orangehrm.test_cases.LoginTest_TC1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
